### DISCUSSION TITLE
about log file

### DISCUSSION THREAD

**Discussion by RyanCode** [FreeRDP/FreeRDP#8813] [Q&A] [answered]
I have referenced a freerdp client in my application. How do I output freerdp logs to a specified file? I tried WLOG_ APPENDER, but it doesn't work

**Comment by kimcoldwater** [comment, marked as answer]
`WLOG_FILEAPPENDER_OUTPUT_FILE_NAME = [ filename ] WLOG_FILEAPPENDER_OUTPUT_FILE_PATH= [ directory ]`




Ex)

`WLOG_APPENDER=file WLOG_FILEAPPENDER_OUTPUT_FILE_NAME=output.log WLOG_FILEAPPENDER_OUTPUT_FILE_PATH=/tmp xfreerdp /v:....
`

**Reply by RyanCode** [reply]
I referenced freerdp in other applications, LDFLAGS -lfreerdp2 -lwinpr2.These environment variables have no effect

**Reply by akallabeth** [reply]
@RyanCode which platform? doing that in some projects, sounds like a bug on your end. (I think `UWP` might be the only exception, that one is not really well  supported)

**Reply by RyanCode** [reply]
centos 7.9. 
this is my shell code
```shell
WLOG_APPENDER="FILE" WLOG_LEVEL="DEBUG" WLOG_FILEAPPENDER_OUTPUT_FILE_PATH="/tmp/freerdp" WLOG_FILEAPPENDER_OUTPUT_FILE_NAME="freerdp.log" ./guacd start -Ldebug
```
when I open a rdp session.there is no freerdp log file in path /tmp/freerdp

**Reply by hardening** [reply]
Might be that the guacamole server does its own setup of WLog, in my memory it does, but you may check the code to be sure.

**Reply by RyanCode** [reply]
Okay, Thank you
